book/history.md

```diff
@@ -11,26 +11,27 @@ exhaustive;[^sgml] the focus is the key events and ideas that led to the web,
 and the goals and motivations of its inventors.
 
 [^sgml]: For example, there is nothing much about
-[SGML](https://en.wikipedia.org/wiki/Standard_Generalized_Markup_Language) or
-other predecessors to HTML. (Except in this footnote!)
+Standard Generalized Markup Language (
+[SGML](https://en.wikipedia.org/wiki/Standard_Generalized_Markup_Language))
+or other predecessors to HTML. (Except in this footnote!)
 
 The Memex concept
 =================
 
 ::: {.center}
-![Figure 1: The original publication of *As We May Think*[^memex]](im/memex.jpg)
+![Figure 1: The original publication of "As We May Think". ([Dunkoman][dunkoman] from [Wikipedia][the-memex], [CC BY 2.0][cc-by-2].)](im/memex.jpg)
```

Comment:
Does this render correctly in HTML?

For other chapters I just ignored the "inline the copyright" edits as a difference between LTML and book outputs.

Reply:
Oh boy, OK. I'll check if they look OK and we probably need to standardize.

Reply:
It does look fine in both web and book. Let's merge this and standardize in another PR.